cmd/juju/model/destroy.go

```diff
@@ -432,7 +441,13 @@ func waitForModelDestroyed(
 				// model has been destroyed successfully.
 				return nil
 			}
-			ctx.Infof(formatDestroyModelInfo(data) + "...")
+			msg := formatDestroyModelInfo(data)
+			if reported == msg {
+				fmt.Fprint(ctx.Stderr, ".")
```

Comment:
Out of interest, does this auto wrap if this get's too long?

Reply:
Yeah, I noticed when I removed a CDK model recently that there were hundreds of repeated lines.

Reply:
Interesting question... I am happy to wrap but it does not seem like I can easily determine terminal width or any other parameters to customize wrapping based on the user client. This means that I'd have to arbitrary wrap at a reasonable string length... I am picking 80 \o/ At 80 characters per line, I'd be wrapping to the next one.